01-gabbar/configs/00-preview/space.yaml

```diff
@@ -3,11 +3,8 @@ kind: Space
 metadata:
   name: gabbar-pr
   labels:
-    team: gabbar
-    kind: pr
```

Comment:
this is required as this label is in SD

Reply:
I want to replace them with env.stakater.com/kind

Reply:
make sense. need to change in stakater gitops first.

Reply:
for now just add it as an extra one